EnglishFilter/sections/antiadblock.txt

```diff
@@ -447,6 +447,9 @@ smallseotools.com#%#//scriptlet("set-constant", "isAdBlockActive", "false")
 @@||smallseotools.com^$generichide,badfilter
 ! distro.tv - antiadblock
 ||distro.tv/js/blocker.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/58355
+@@||faupto.com/freeltc/libs/show_ads.js
+faupto.com#%#//scriptlet("set-constant", "show_ads_gr8_lite", "true")
```

Comment:
Is it related to this issue?

Reply:
Yes, after captcha sometimes we can see adblock detection in the iframe loaded from faupto.com

Reply:
I thought you had to fix everything else that got caught - not only reported trouble.

Reply:
got it